src/stactools/palsar/commands.py

```diff
@@ -22,14 +22,16 @@ def palsar():
         "create-collection",
         short_help="Creates a STAC collection",
     )
+    @click.argument("product")
     @click.argument("destination")
-    def create_collection_command(destination: str):
+    def create_collection_command(product: str, destination: str):
         """Creates a STAC Collection
 
         Args:
+            product (str): MOS or FNF Collection type
```

Comment:
Is it possible to use an enum type or equivalent here to make the code a bit more robust?

Reply:
@tracetechnical No objections to trying that, did you want to outline or make a PR?

Reply:
Im not entirely sure how to go about it in Python. The major possible blocker to this is whether or not "that command line library in python that I cant remeber the name of" supports enum type arguments.

The intent would be to provide basic input constraint to make the code a bit more robust. Thinking "new user passes some absolute crap into this param" sort of usecase.

If this isn't commonly done when using this library then thats ok too I guess.

Reply:
This what you're thinking https://click.palletsprojects.com/en/8.0.x/options/#choice-options ?

Reply:
Not sure options will work as this is a required arg. Note the code assumes a value of MOS if none given (the actual function).